server/master/src/main/java/org/apache/accumulo/master/TabletGroupWatcher.java

```diff
@@ -669,7 +670,7 @@ private void deleteTablets(MergeInfo info) throws AccumuloException {
             + Path.SEPARATOR + extent.getTableId() + Constants.DEFAULT_TABLET_LOCATION;
         MetadataTableUtil.addTablet(
             new KeyExtent(extent.getTableId(), null, extent.getPrevEndRow()), tdir,
-            master.getContext(), timeType, this.master.masterLock);
+            master.getContext(), metadataTime.getType(), this.master.masterLock);
```

Comment:
There is a difference in the code here in that metadataTime could possibly be null.  However I think its ok, because I am thinking it should not be null because tablets should always have time type.  Also the old code was doing something really strange with `\0`

Reply:
Yes.  I was not happy with assigning null but that is what the previous code was doing although it was obviously always assigned something because there are no null char checks down the line.  It could be assigned the default value of MILLIS.

Reply:
Thinking about it a bit more, I think your change is good because it would fail with an NPE if not set.  I think the expectation is that every tablet should have this column.